builder/linode/config.go

```diff
@@ -40,24 +44,26 @@ type Config struct {
 	ctx                 interpolate.Context
 	Comm                communicator.Config `mapstructure:",squash"`
 
-	Interfaces         []Interface       `mapstructure:"interface"`
+	Interfaces         []Interface       `mapstructure:"interface" required:"false"`
```

Comment:
Just curious, what does the `required:"false"` here do? Are these fields required by default?

Reply:
I believe that it's just a stylistic change to explicitly mark them as optional attributes.